server/src/main/java/io/crate/replication/logical/action/PublicationsStateAction.java

```diff
@@ -227,22 +228,29 @@ static List<RelationName> resolveRelationsNames(Publication publication,
                 relationNames = publication.tables();
             }
             return relationNames.stream()
-                .filter(relationName -> userCanPublish(relationName, publicationOwner))
-                .filter(relationName -> subscriberCanRead(relationName, subscribedUser))
+                .filter(relationName -> userCanPublish(publicationName, relationName, publicationOwner))
+                .filter(relationName -> subscriberCanRead(publicationName, relationName, subscribedUser))
                 .toList();
         }
 
-        private static boolean subscriberCanRead(RelationName relationName, User subscriber) {
-            return subscriber.hasPrivilege(Privilege.Type.DQL, Privilege.Clazz.TABLE, relationName.fqn(), Schemas.DOC_SCHEMA_NAME);
+        private static boolean subscriberCanRead(String publicationName, RelationName relationName, User subscriber) {
+            boolean canRead = subscriber.hasPrivilege(Privilege.Type.DQL, Privilege.Clazz.TABLE, relationName.fqn(), Schemas.DOC_SCHEMA_NAME);
+            if (canRead == false) {
+                LOGGER.warn("User {} subscribed to the publication {} doesn't have DQL privilege on the table {}, this table will not be replicated.",
```

Comment:
Spent quite some time when forgot to give `DQL` permission in tests - probably it worth to warn users about skipping a table in `ALL TABLES` publication.